compiler/node.h

```diff
@@ -71,12 +76,21 @@ class Node : public NodeBase {
 
     std::string ToString() const;
 
+    const OpsetList& opset_imports() const {
+        return opset_import_;
+    }
+
+    int OpVersion() const;
+
+    bool ValidateWithSchema(const OpsetList& opsets = {}, std::string* message = nullptr) const;
+
 private:
     std::vector<Value*> inputs_;
     std::vector<Value*> outputs_;
     std::string name_;
     std::string domain_;
     std::string doc_string_;
+    OpsetList opset_import_;
```

Comment:
I want all non-ephemeral states of these classes to be serialized in ONNX so that we can stop and resume graph transformation at any stage. Could you introduce custom attributes to serialize this info? Probably:

```
CHAINER_COMPILER_GLOBAL_ATTRS = attr_sets(chainer_order=-1, chainer_fusion_group=0, chainer_onnx_domain=[str], chainer_onnx_version=[int])
```

? (I found a typo in `CHAINER_COMPILERX_GLOBAL_ATTRS`)

Reply:
Making it to attribute sounds nice :)